### DISCUSSION TITLE
Channel issue on TeamTalk 5.8.1

### DISCUSSION THREAD

**Discussion by computertech1012** [BearWare/TeamTalk5#1158] [General]
Hey guys, so in TeamTalk 5.8.1, if you use the join channel optional field to specify a channel name and the channel password, the password does not get saved if the connection is lost. Sometimes it might take a few attempts, but you could reproduce this issue by going to the server dialogue, and in the join channel optional field, specify a channel name and a channel password, once you specify that, connect to it, then break the connection. If it's a temporary channel, it will re-create the channel without a password. However, if it is a permanent channel, it should automatically join the channel however, it does not. Instead, it will display an error message saying incorrect channel password. It seems like it's not saving passwords. This is happening in the qt client. Is anybody else able to help reproduce this issue? A friend brought this up to me, and I decided to create a discussion group about it. Also, sometimes it might take two or three different times in order to reproduce.

**Comment by bear101** [comment]
I would recommend you find a reliable way to reproduce the issue. I've tried several times and was unable to reproduce it.
